libraries/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -193,6 +194,11 @@ export abstract class BaseInstallManager {
       statePropertiesToIgnore: optionsToIgnore
     }));
 
+    const rushConnect: RushConnect = RushConnect.loadFromLinkStateFile(this.rushConfiguration);
+    const isNodeModulesOverWritten: boolean = await rushConnect.isSubspaceDependencyLinkedAsync(
+      subspace.subspaceName
+    );
```

Comment:
Consider gating this (and the new commands) behind a flag in `common/config/rush/experiments.json`.